webapp/channels/src/components/menu/menu.tsx

```diff
@@ -232,9 +238,10 @@ export function Menu(props: Props) {
                         exit: MENU_CLOSE_ANIMATION_DURATION,
                     },
                 }}
-                width={props.menu.width}
             >
-                {props.children}
+                <MenuContext.Provider value={{close: closeMenu, isOpen: isMenuOpen}}>
```

Comment:
I think we may want to use a `useMemo` to construct the value object

Reply:
Done ✅